src/inmanta/server/services/orchestrationservice.py

```diff
@@ -134,77 +160,74 @@ def pair_resources_partial_update_to_old_version(
 
     async def _get_old_resources(
```

Comment:
I think this method and `_merge_resources` could still benefit from a docstring. How/where does this method get the old resources? What does `_merge_resources` do and return?